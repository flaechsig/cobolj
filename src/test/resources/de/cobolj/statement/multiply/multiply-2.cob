IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-NUMBER-RESULT-1 PIC 9(3) VALUE 60.
   01 WS-NUMBER-RESULT-2 PIC 9(2) VALUE 60.

PROCEDURE DIVISION.
   MULTIPLY 2 BY WS-NUMBER-RESULT-1 GIVING WS-NUMBER-RESULT-1  WS-NUMBER-RESULT-2
   ON SIZE ERROR DISPLAY "SPEICHERUEBERLAUF".
   DISPLAY WS-NUMBER-RESULT-1.
   DISPLAY WS-NUMBER-RESULT-2.
STOP RUN.
