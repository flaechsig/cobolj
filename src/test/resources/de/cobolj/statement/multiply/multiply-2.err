Multiplikation eines Wertes auf mehrere Ziele mit SpeicherÃ¼berlauf