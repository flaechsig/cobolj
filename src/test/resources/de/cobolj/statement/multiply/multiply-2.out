SPEICHERUEBERLAUF
120
60
